### PR TITLE
#14

### PR DESCRIPTION
**Pull request by apocarteres**

### CODE REVIEW

**Comment by dmarkov** [comment]
Let me find a reviewer for this pull request, thanks for submitting it

**Comment by dmarkov** [comment]
@krzyk please review this one

File: README.md [hunk 1]
```diff
@@ -22,6 +22,16 @@ Use it in your `site.xml`, as explained in
 </project>
 ```
 
+Don't forget to install `fontcustom` Ruby gem:
+
+On Linux/OS X it would be something like:
+
+```
+$ sudo gem install fontcustom
```
Comment (krzyk) [inline README.md]: Remove `sudo`

**Comment by krzyk** [comment]
@apocarteres actually this is required only for development not when using this skin. Moreover font custom has been already added to the README 4 days ago,  so I would close this PR

**Comment by apocarteres** [comment]
@krzyk sure. thx

**Comment by dmarkov** [comment]
@krzyk **15 mins** added to your account (payment number `51519058`), many thanks for your contribution!. +15 added to your rating, current score is: [+3826](http://www.netbout.com/b/35035?open=rating)

**Pull request closed without merging**